orc8r/cloud/docker/.env

```diff
@@ -9,4 +9,3 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-COMPOSE_PROJECT_NAME=orc8r
```

Comment:
Does removing these variables have unintended side effects? E.g. something like this could be affected https://github.com/magma/magma/blob/master/orc8r/tools/docker/publish.sh#L50

Reply:
The `COMPOSE_PROJECT_NAME` is used as a prefix to publish FEG and CWF docker images. Instead of deleting the variables and letting docker do its naming magic, I used the (at some parts already used) `DOCKER_REGISTRY` variable to mitigate the switch to the "new" naming convention. The publishing script now still has it's prefix variable.